civihr_default_theme/scss/components/_button.scss

```diff
@@ -1,8 +1,16 @@
+.btn {
+  box-shadow: none !important;
```

Comment:
i don't see no explanation for this in the PR. Why is this necessary? I cannot see any `box-shadow` applied on buttons in the SSP

Reply:
@AkA84 updated the description and the commit. https://github.com/compucorp/civihr-employee-portal-theme/pull/328/commits/b4ee86a531ab577e62c44f53d07dc53ef5efcef0#diff-df5d683b3db579634d916ca4a287a0dfR1